src/main/resources/application.properties

```diff
@@ -1,2 +1,5 @@
 spring.main.web-environment=false
-spring.data.mongodb.database=submissions
\ No newline at end of file
+spring.data.mongodb.database=submissions
```

Comment:
I think we do not need the 'spring.data.mongodb.database' properties here. It is defined in the secrets repo.

Reply:
Agreed